### PR TITLE
checked None for senses

### PR DESCRIPTION
**Pull request by udkumar**

### CODE REVIEW

File: agmt/main.py [hunk 1]
```diff
@@ -714,7 +714,7 @@ def updateProjectTokenTranslations():
 			if senses == rst[2] and translation == rst[1]:
 				return '{"success":false, "message":"No New change. This data has already been saved"}'
 			dbSenses = []
-			if rst[2] != "":
+			if rst[2] != None or rst[2] != "":
```
Comment (kavitharaju) [inline agmt/main.py:717]: shouldn't the connective be AND ?

File: agmt/main.py [hunk 2]
```diff
@@ -807,8 +808,10 @@ def bulkUpdateProjectTokenTranslations():
 							userId, senses))
 				else:
 					dbSenses = []
-					if rst[2] != "":
+					if rst[2] != None or rst[2] != "":
```
Comment (kavitharaju) [inline agmt/main.py:811]: shouldn't it be AND?

**Review by kavitharaju — COMMENTED** [review]
Please confirm which should connective be used there? AND or OR?
I feel you need AND as both the condition should be satisfied.

**Review by kavitharaju — COMMENTED** [review]
What about the check in `bulkUpdateProjectTokenTranslations()` line number `812` ?

**Review by kavitharaju — APPROVED** [review]

**Review by udkumar — COMMENTED** [review]
We have tested with both but working with None only.... so updated the code

**Review by Joel-C-Johnson — APPROVED** [review]

**Pull request merged**